### ISSUE TITLE
Document how to use grammar specifications different from packaged ones

### ISSUE THREAD

**Issue by bast** [dev-cafe/parselglossy#79]
I think the grammar definitions should be part of the tutorials, not part of the core library.

**Comment by robertodr** [comment]
Yes, once this project becomes a generator we can "synthesize" input handler and documentation from template and grammar that live entirely within the client.

**Comment by robertodr** [comment]
To be decided with #84

**Comment by robertodr** [comment]
This is solved, I think. The generator is the recommended way to use parselglossy and the grammar can be defined in whatever way (use of pyparsing not required!) as long as the result of that step is a dictionary stored in a variable called `ir`. This is documented and I suggest to close this, unless you think more documentation would help clarify this point.

**Comment by bast** [comment]
I think what is left to do is to document how to use own grammar or how to change the grammar. We can also rename the issue or start a new one.
